src/main/java/org/folio/oaipmh/helpers/records/RecordMetadataManager.java

```diff
@@ -160,7 +187,7 @@ private void updateFieldsWithItemElectronicAccessField(JsonObject itemData, List
     String name = electronicAccess.getString(NAME);
     String key = StringUtils.isNotEmpty(name) ? name : EMPTY;
     String indicatorsInString = indicatorsMap.get(key);
-    if(indicatorsInString!=null) {
+    if (indicatorsInString != null) {
```

Comment:
Possibly this can be relpaces by simpler case to avoid conditional operator usage, I man something like this:

return Arrays.asList(Optional.ofNullable(indicatorsInString ).orElse(StringUtils.EMPTY).split(","));

Reply:
All existing code can't be fixed in one pull request, can it?